### PR TITLE
tpm_device: add swtpm debug logging test

### PR DESCRIPTION
**Pull request by yanqzhan**
Add two classical values '5','20' test, and a negative '-1' test. Swtpm/libtpms debugging is very much non-stable/non-contract thing according to developer, our QE can first check sth different from non-setting, not sure the obvious difference between each level.

### CODE REVIEW

**Comment by yanqzhan** [comment]
Depends on https://github.com/avocado-framework/avocado-vt/pull/4026

**Comment by yanqzhan** [comment]
(1/3) type_specific.io-github-autotest-libvirt.virtual_devices.tpm_device.normal_test.tpm-crb_model.emulator.plain.pcrbank_default.debug.lvl_5: PASS (39.59 s)
  (2/3) type_specific.io-github-autotest-libvirt.virtual_devices.tpm_device.normal_test.tpm-crb_model.emulator.plain.pcrbank_default.debug.lvl_20: PASS (39.90 s)
 (3/3) type_specific.io-github-autotest-libvirt.virtual_devices.tpm_device.negative_test.emulator.invalid_debug: PASS (9.32 s)

File: libvirt/tests/cfg/virtual_device/tpm_device.cfg [hunk 1]
```diff
@@ -39,6 +39,15 @@
...
                             backend_version = 'none'
+                        - debug:
+                            func_supported_since_libvirt_ver = (10, 6, 0)
+                            only encrypted 
+                            only pcrbank_default
+                            variants:
+                                - lvl_5:
+                                  backend_debug = '5'
+                                - lvl_20:
+                                  backend_debug = '20'
```
Comment (cliping) [inline libvirt/tests/cfg/virtual_device/tpm_device.cfg:50]: indent on lines 48 and 50 is wrong.

File: libvirt/tests/cfg/virtual_device/tpm_device.cfg [hunk 2]
```diff
@@ -209,6 +218,11 @@
                                 - version_textdefault:
                                     backend_version = 'default'
                                     xml_errmsg = "Invalid value for attribute 'version' in element 'backend': 'default'"
+                        - invalid_debug:
+                           func_supported_since_libvirt_ver = (10, 6, 0)
+                           backend_debug = '-1'
+                           backend_version = '2.0'
+                           xml_errmsg = "Invalid.*'debug' in element 'backend'.*Expected non-negative integer value"
```
Comment (cliping) [inline libvirt/tests/cfg/virtual_device/tpm_device.cfg:225]: From 222 to 225 lines, indent issue.

**Review by cliping — COMMENTED** [review]
Others LGTM

File: libvirt/tests/cfg/virtual_device/tpm_device.cfg (hunk 1 above)
Comment (yanqzhan) [reply libvirt/tests/cfg/virtual_device/tpm_device.cfg]: Done. Thx.

File: libvirt/tests/cfg/virtual_device/tpm_device.cfg (hunk 2 above)
Comment (yanqzhan) [reply libvirt/tests/cfg/virtual_device/tpm_device.cfg]: Done. Thx.

**Review by cliping — APPROVED** [review]

**Review by chloerh — APPROVED** [review]
